Lombiq.Privacy.csproj

```diff
@@ -1,9 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
+  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props" />
+
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
     <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**;Tests\**</DefaultItemExcludes>
+    <NodeJsExtensionsCreateStylelintConfigurationFile>false</NodeJsExtensionsCreateStylelintConfigurationFile>

```

Comment:
This is something we'll need to do in each csproj of OSOCE. Any better ideas? Setting the property globally in the Directory.Build.props file, and then during NuGet publishing in GitHub Action would be better (though it seems that unless we care about an unused stylelingrc file being created this shouldn't matter, because it's not embedded in the NuGet anyway, right?).

Or perhaps having `false` as default would be better, given that we'll need to set that in every solution of hours even though it might be better as `true` for users outside of Lombiq (though I'm not sure).

Reply:
The problem is that without a configuration object, Stylelint will throw an error. That's why, the default needs to be true for all projects that simply want to use NE, or at least so the initial setup is as simple as possible.

Setting it in the _Directory.Build.props_ is a very good idea and would solve the problem. For the NuGet, we don't need to care, because we don't lint anyways, and yes, the file is not included in the package.

Reply:
Should be good now.